README.md

```diff
@@ -38,7 +53,60 @@ npm install
 
 ## Usage
 
-tbd
+```js
+const {handleTokenExchange} = require('@digitalbazaar/oauth2-token-endpoint-handler');
+
+app.post('/token',
+  // ... make sure to mount url-encoded body parser such as express.urlencoded()
+  handleTokenExchange({
+    baseUrl: 'https://as.example.com',
+    defaults: {
+      // (Optional) token expiration in seconds. Defaults to 2 weeks
+      defaultMaxAge: 1209600
+    },
+    logger: console,
+    getClient: async ({clientId}) => {
+      // load client from the db, then:
+      return {client};
+    },
+    issue: async ({client, resource, scope, body, defaultMaxAge}) => {
```

Comment:
```suggestion
    async issue({client, resource, scope, body, defaultMaxAge}) {
```

Reply:
@dlongley I'm not sure the suggested change properly communicates to the reader that those are callbacks.

Reply:
Why not?

Reply:
@dlongley because it looks like a function declaration in a module, rather than the usual callback syntax. 
I am -1 on this suggestion; it made me pause and re-read the code several times to even figure out what you meant.